src/loader.c

```diff
@@ -134,7 +135,7 @@ static const image_decoder decoders[] = {
 #ifdef HAVE_LIBTIFF
     &LOADER_FUNCTION(tiff),
 #endif
-    &LOADER_FUNCTION(qoi),  &LOADER_FUNCTION(tga),
+    &LOADER_FUNCTION(qoi),  &LOADER_FUNCTION(tga), &LOADER_FUNCTION(ff)
```

Comment:
Please move ff decoder in front of tga (tga doesn't have any magic id and should be the last decoder).